Sources/TelemetryClient/LogHandler.swift

```diff
@@ -32,6 +44,16 @@ public struct LogHandler {
         }
     }
 
+    @available(iOS 14.0, macOS 11.0, watchOS 7.0, tvOS 14.0, *)
+    public static var oslog = { logLevel in
+        LogHandler(logLevel: logLevel) { level, message in
+            Logger(
+                subsystem: "TelemetryDeck",
+                category: "LogHandler"
+            ).log(level: logLevel.osLogLevel, "\(message, privacy: .public)")
```

Comment:
I believe the specifier `privacy: .public` is redundant here since during logging the privacy level is `public` by default.